### PR TITLE
Update manifest.toml

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -41,18 +41,18 @@ yunohost service add "$app" --description="Radio Broadcast & Automation Platform
...
-#  	--listen-port $port \
-#  	--no-setup-postgresql \
-#  	https://$domain
-#popd
-
 pushd "$install_dir"
-	./install -fiap
+	./install \
+	--user $app \
+	--storage-path $data_dir
```
Comment (Tagadda) [inline scripts/install]:
```suggestion
	--storage-path $data_dir \
```

**Comment by ericgaspar** [comment]
!testme

**Comment by OniriCorpe** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!bullseyetestme

**Pull request merged**